PublicMapping/districtbuilder
# Update map view for evaluate mode

## Overview

- Add polylabel library
- Add geojson flatten
- Remove highlighted / selected district in evaluate mode

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/116895287-0728ef00-ac01-11eb-8ba7-cec76eeb3e65.png)

## Notes

I'm struggling a bit with how to make the place labels in the tiles not displayed in this view. Is this a hard requirement here?

## Testing Instructions

- Open a project and click Evaluate
- Expect: Evaluate mode looks like the above screenshot
- Close evaluate mode
- Expect: Project map is rendered in the same state before toggling evaluate mode

Closes #604
